In [2]:
import sys
print(sys.version)

3.13.9 | packaged by Anaconda, Inc. | (main, Oct 21 2025, 19:11:29) [Clang 20.1.8 ]


In [4]:
!conda create -n agentic-ai python=3.11 -y
!conda activate agentic-ai

Jupyter detected...
2 channel Terms of Service accepted
Channels:
 - defaults
Platform: osx-arm64
Solving environment: done


==> WARNING: A newer version of conda exists. <==
    current version: 26.1.1
    latest version: 26.5.3

Please update conda by running

    $ conda update -n base -c defaults conda



## Package Plan ##

  environment location: /Users/sriram/anaconda3/envs/agentic-ai

  added / updated specs:
    - python=3.11


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    packaging-26.0             |  py311hca03da5_0         196 KB
    python-3.11.15             |       h478e877_1        14.8 MB
    setuptools-82.0.1          |  py311hca03da5_0         1.6 MB
    wheel-0.46.3               |  py311hca03da5_0          71 KB
    ------------------------------------------------------------
                                           Total:        16.6 MB

The following NEW packa

In [ ]:
!pip install -U \
    langchain \
    langchain-community \
    langchain-groq \
    langgraph \
    fastmcp \
    mcp \
    nest_asyncio

  Using cached langchain_community-0.4.2-py3-none-any.whl.metadata (3.4 kB)
  Using cached fastmcp-3.4.2-py3-none-any.whl.metadata (8.5 kB)
  Using cached mcp-1.28.0-py3-none-any.whl.metadata (9.4 kB)
  Using cached langchain_core-1.4.7-py3-none-any.whl.metadata (4.5 kB)
  Using cached langgraph_checkpoint-4.1.1-py3-none-any.whl.metadata (5.2 kB)
  Using cached langgraph_prebuilt-1.1.0-py3-none-any.whl.metadata (5.2 kB)
  Using cached xxhash-3.7.0-cp313-cp313-macosx_11_0_arm64.whl.metadata (13 kB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 554.9/554.9 kB 20.7 MB/s  0:00:00
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 562.7/562.7 kB 9.4 MB/s  0:00:00
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 2.4/2.4 MB 4.7 MB/s  0:00:005.3 MB/s eta 0:00:01:01
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.0/1.0 MB 5.3 MB/s  0:00:004.9 MB/s eta 0:00:01
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 749.2/749.2 kB 5.1 MB/s  0:00:00
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.3/1.3 MB 4.1 MB/s  0:00:00m 4.

Groq has fast inference model

5.4 opena AI model need to use don't use before it

claude 4.7

prompt cache

claw code from ultraworkers for harness

OCR 

MCP setup need to do at own servers
MCP TOOL
MCP Server

Environment as Service (Ex : google colab with RAM and Disk)

skills.md

we should have a logical agents

skills market place
ex : VoltAgent (awesome-desing-md)

heyGen (Ai text to video)








